sdk/inc/azure/core/az_version.h

```diff
@@ -29,6 +29,7 @@
 #define AZ_SDK_VERSION_PATCH 0
 
 /// Optional pre-release identifier. SDK is in a pre-release state when present.
-#define AZ_SDK_VERSION_PRERELEASE "beta.2"
+#define AZ_SDK_VERSION_PRERELEASE
+#undef AZ_SDK_VERSION_PRERELEASE
```

Comment:
why the undef?

Reply:
I don't fully understand/remember the detail, but it has to do with how we we'd like to have the doxygen docs rendered.

See this thread and discussion for why we have it (based on @antkmsft's recommendation):
https://github.com/Azure/azure-sdk-for-c/pull/1336#discussion_r492428630

This is the pattern we typically follow, whenever doing a GA release.

For example:
https://github.com/Azure/azure-sdk-for-c/pull/1899